nvtabular/ops/normalize.py

```diff
@@ -94,16 +94,19 @@ def supports(self):
         )
 
     def clear(self):
+        """Clear previously recorded stats"""
```

Comment:
Can we just use the docstrings from the parent class? I'd rather not maintain two copies of the same content just to make interrogate happy